bindings/wasm/examples/public/examples.js

```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/** @type {import("../../manifold-encapsulated-types")["Manifold"]} */
+const Manifold = globalThis.Manifold;
```

Comment:
What does this do?

Reply:
I added those two lines to get the TypeScript typechecking working in this .js file. TypeScript allows you to specify types in .js files via the JSDoc syntax: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#type

This is purely for dev convenience, it doesn't impact the ManifoldCAD page at all, since the types are injected directly to the editor there.

Reply:
Nice, I didn't know that!